### ISSUE TITLE
[Adaptive Cards - iOS - Action.OpenURL.Json]: Voice over is conveying the 'This Card's action will open a URL' label information twice. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8422]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'Action.OpenURL.Json' and activate it.
7. Card will appear.
8. Navigate to the **''This Card's action will open a URL''** label and observe the issue.

**Actual Result:**
While navigating to the 'This Card's action will open a URL' label, screen reader is conveying the information twice.

**Expected Result:**
Screen Reader should convey the label information only one time and it should announce the information as ''This Card's action will open a URL'' label.

**Note:**
This issue is repro throughout the application.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228520608-92dc1c84-7643-44f6-a868-2eed5945d3bf.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue here as of now this issue is being tracked using below URL.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/36
